web-ui/src/main/resources/catalog/views/default/directives/directive.js

```diff
@@ -364,4 +364,37 @@
       };
     }
   ]);
+
+  module.directive('gnAppUrlAccessibilityCheck', ['$http', function($http) {
+    return {
+      restrict: 'A',
+      scope: {
+        isAccessible: '='
+      },
+      link: function(scope, element, attrs) {
+        $http.get(attrs.gnAppUrlAccessibilityCheck)
```

Comment:
using the HEAD verb would be more appropriate here, otherwise the client may end up downloading a lot of stuff

Reply:
I originally used HEAD but then I did not like seen all the error message in the connection logs so I added the option to read the results from the body.  If we do a HEAD term then we cannot use the body to check for text like "false, 0,,,"

I'm not sure if it is considered bad practice to check the body text or not?  In our organization, we do have a couple API's that simply return an HTTP code 200 with "true" or "false" in the response body.

I can remove the body check and change it to a head request if that is best practice.